tests/metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,95 @@
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Port number where the server will run
+port=5201:5201
+# Image name
+image=gabyct/network
+# Measurement time (seconds)
+time=5
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+# This script will perform all the measurements using a local setup using iperf3
+
+function iperf3_bandwidth {
+	setup
+	bandwidth_result=$(mktemp)
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "iperf3 -p ${port} -s" > /dev/null && \
+	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
+	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "iperf3 -c ${server_address} -t ${time}" > "$bandwidth_result"
+	total_bandwidth=`cat $bandwidth_result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}'`
+	$DOCKER_EXE rm -f iperf3-server > /dev/null
+	rm -f $bandwidth_result
+}
+
+function iperf3_jitter {
+	setup
+	jitter_result=$(mktemp)
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "iperf3 -s -V" > /dev/null && \
+	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
+	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "iperf3 -c ${server_address} -u -t ${time}"  > "$jitter_result"
+	total_jitter=`cat $jitter_result | tail -n 4 | head -1 | awk '{print $(NF-4), $(NF-3)}'`
+	$DOCKER_EXE rm -f iperf3-server > /dev/null
+	rm -f $jitter_result
+}
+
+function iperf3_bidirectional_bandwidth_remote_local {
+	setup
+	bidirectional_bandwidth_result=$(mktemp)
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "iperf3 -p ${port} -s" > /dev/null && \
+	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
+	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "iperf3 -c ${server_address} -d -t ${time}" > "$bidirectional_bandwidth_result"
+	total_bidirectional_bandwidth=`cat $bidirectional_bandwidth_result | tail -n 3 | head -1 |  awk '{print $(NF-2), $(NF-1)}'`
+	$DOCKER_EXE rm -f iperf3-server > /dev/null
+	rm -f $bidirectional_bandwidth_result
+}
+
+function iperf3_bidirectional_bandwidth_local_remote {
+	setup
+	bidirectional_bandwidth_result=$(mktemp)
+	$DOCKER_EXE run -d --name=iperf3-server ${image} bash -c "iperf3 -p ${port} -s" > /dev/null && \
+	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
+	$DOCKER_EXE run -ti --rm --name=iperf3-client ${image} bash -c "iperf3 -c ${server_address} -d -t ${time}" > "$bidirectional_bandwidth_result"
+	total_bidirectional_bandwidth=`cat $bidirectional_bandwidth_result | tail -n 4 | head -1 | awk '{print $(NF-3), $(NF-2)}'`
+	$DOCKER_EXE rm -f iperf3-server > /dev/null
+	rm -f $bidirectional_bandwidth_result
+}
+
+echo "Currently this script is not working with CC (see https://github.com/01org/cc-oci-runtime/issues/152)"
```

Comment:
@GabyCT How about running these tests with using ramfs for /tmp as mentioned in the issue?